### DISCUSSION TITLE
CREALITY FALCON2 40W

### DISCUSSION THREAD

**Discussion by Rocket3man61** [arkypita/LaserGRBL#2383] [General]
Hi all. 
I'm fairly new to the laser grill scene. It's a great program and soooo easy to use with great results. 

My question is this, how can I cut the same image multiple times without having to move the laser manually each time, it seems I can only cut one at a time.
Many thanks for any help.

**Comment by Bhowell52** [comment]
Are you wanting to make several passes on the same piece of wood or different pieces of wood?
same wood there is a drop down on the upper left part of grbl that ask how many passes you want.
If you are changing wood I have a scale on my table that I place my wood and I use this when I know im going to be making multiple pieces.
hope this worked for you

**Comment by Rocket3man61** [comment]
I'm making lids for the scented candles my wife makes, but I want to make
more than just one at a time, I just wondered if there's something like
copy and paste on laser grbl

**Comment by Tribo-2dot0** [comment]
Monte o design em Inkscape, use copiar, colar e use as ferramentas de alinhamento e distribuição... salve o .svg e abra-o em LaserGrbl.

**Comment by Rocket3man61** [comment]
Gracias, lo intentaré
